integrationTests/cypress/integration/legal_urls.feature

```diff
@@ -0,0 +1,6 @@
+Feature: Simple test to check the forgotten password functionality of cas
+
+  Scenario: the user successfully logs into the CES with admin credentials and CAS shows the profile page
+    Given the user is currently not logged in the CES
+    Given the browser shows the cas login page
```

Comment:
Gerkin tests without Given/When/Then steps seem kinda awkward. Maybe change the step `Given the browser shows the CAS login page` into `When the user visits the CAS login page`

Please spell CAS big as it is an abbreviation for a name.